apps/scan/backend/src/server.ts

```diff
@@ -35,27 +25,13 @@ export function start({
   usbDrive,
   workspace,
 }: StartOptions): void {
-  /* c8 ignore start */
-  const resolvedAuth =
-    auth ??
```

Comment:
What was the use case for supporting `undefined` auth?

Reply:
I think for testing/dependency injection? It doesn't appear to be needed at any rate.